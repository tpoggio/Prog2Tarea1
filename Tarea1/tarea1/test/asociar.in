esClave 15
asociar 15 5.2
esClave 15
asociar 95 24.0
esClave 95
asociar 85 2.0
asociar 65 5.0
asociar 25 67.0
asociar 35 -12.0
asociar 75 2.5
asociar 45 2.1
asociar 55 -24.0
# clave repetida; no tiene efecto
asociar 85 32.0
# no esta llena
asociar 5 32.0
esClave 5
# esta llena
asociar 10 10.0
esClave 10
Fin
