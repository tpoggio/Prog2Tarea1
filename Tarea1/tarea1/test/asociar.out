1>15 no es clave. 
2>Asociar.
3>15 es clave. 
4>Asociar.
5>95 es clave. 
6>Asociar.
7>Asociar.
8>Asociar.
9>Asociar.
10>Asociar.
11>Asociar.
12>Asociar.
13>#  clave repetida; no tiene efecto.
14>Asociar.
15>#  no esta llena.
16>Asociar.
17>5 es clave. 
18>#  esta llena.
19>Asociar.
20>10 no es clave. 
21>Fin.
